### ISSUE TITLE
Create additional fields within the « CASE » modules

### ISSUE THREAD

**Issue by carolinverset** [hzi-braunschweig/SORMAS-Project#2410]
### Feature Description

Create additional fields within the « CASE » modules to allow the entry of data meant to be transferred from contact tracing level 3 units (organized within regional healthcare agencies) to contact tracing level 2 units (organized within local healthcare insurance institutions).

**→           Implementation of a multiple choices variable « CASE’S NEEDS »  with the following choices :**

- Immediate RT-PCR prescription 
- D+7 RT-PCR prescription 
- Medical absence from work certificate 
- Other(s)

→           **Implementation of a free text comment field “additional information about the case"** allowing furnishing details about the item “other” if selected at the “CASE’S NEED” variable or entering additional information about the investigated case.

**Comment by ChristopherRiedelSymeda** [comment]
@carolinverset There is already an additional information free text field for case at the bottom of the page.

**Comment by carolinverset** [comment]
I agree with you and we have see the free text field on the bottom of the view. Unfortunately, this field is used for another type of information which is imported from on other data health system.  
Insofar as we have to establish a specific feedback to the local healthcare insurance institutions with information about tests or medical work stopping for cases or contacts, so we need to have two related/linked fields :  multiple choices variable « CASE’S NEEDS » or “contact’s need” AND an attached free text field  allowing to complete this information.  
What are you thinking about ?

**Comment by carolinverset** [comment]
@ChristopherRiedelSymeda 
sorry, the above question was for you.

**Comment by bernardsilenou** [comment]
@carolinverset  Then list of variables you suggested consisted of clinical (PCR) and non clinical (medical absence from work) variables. I will suggest that we do not put these categories in one variable.
In addition, the "therapy" section of SROAMAS was designed to document prescriptions and other medical needs that the patient need. Adding prescriptions to the case data as an attribute is some how mixing stuffs. Is there a reason why we cannot use the therapy part to document the clinical variables that you ask?

I suggest we add the variable "medical absence from work" to the case data and documents all clinical variables  under therapy. We may adjust the therapy part is needed. What do you think please?

**Comment by bernardsilenou** [comment]
@MateStrysewskeSym we decided with the French team that "Immediate RT-PCR prescription" and "D+7 RT-PCR prescription" are prescriptions for the case to do a PCR test, it is better to add them under therapy. So we can add both under prescription type:
This will be a 2 level variable with forms modified accordingly
level 1 : prescription type = "Laboratory test"  (first figure)
level 2 : prescription details = "list of all lab test we have in the system" (second figure)

![image](https://user-images.githubusercontent.com/12996724/86285258-90882300-bbe4-11ea-891e-139f55e425af.png)

![image](https://user-images.githubusercontent.com/12996724/86285600-3045b100-bbe5-11ea-9414-857e224061b6.png)

@carolinverset  What does D+7 RT-PCR means? is it a second PCR test to be done on day 7 after the first PCR?

**Comment by bernardsilenou** [comment]
With respect to variables "Medical absence from work certificate" and "Other(s)", These are official documents and has nothing to do with the epidemiology of the disease. Similar to variables  like  "quarantine order officially"  "quarantine order verbally"  that we have in the German instance. 
The solution we found is to add the variable "Medical certificate of absence from work" to contacts and cases. (small yes/no/unk radio button)

There is a need for the German authorities to document more official document linked to cases and contacts. This will come in a separate issue with a better solution that might be to create a separate tab for documentation all official documents, photos, etc. linked to the case or contact entity 
We decided that the French team should implement this issue

Whats do u think please?

**Comment by ChristopherRiedelSymeda** [comment]
@carolinverset 
> I agree with you and we have see the free text field on the bottom of the view. Unfortunately, this field is used for another type of information which is imported from on other data health system.
> Insofar as we have to establish a specific feedback to the local healthcare insurance institutions with information about tests or medical work stopping for cases or contacts, so we need to have two related/linked fields : multiple choices variable « CASE’S NEEDS » or “contact’s need” AND an attached free text field allowing to complete this information.
> What are you thinking about ?

I think it should be no problem to add a free text field for further information when selecting "Others". But it should not be called "additional information about the case". This could lead to misunderstandings.

**Comment by bernardsilenou** [comment]
@MateStrysewskeSym I recommended in this issue that we should use therapy tab to document prescriptions for cases, but contacts do not have therapy tab for now. Whats your solution to this please?
1. Add therapy to contacts? (I think Swiss requested this feature)
2. Add prescription of PCR test to contact entity (this is easy but is in conflict this the general design of sormas)
3. Others from you

**Comment by bernardsilenou** [comment]
@carolinverset we had a deeper look today and realized it is not a good option to add D+7 RT-PCR to therapy.  We do not have a fixed solution now for this issue, so I will recommend we think over it again.

Another idea we though of is to add the prescription of D+7 RT-PCR close to the sample because a prescription will lead to a sample being collected and a test result done. This is not still clear on how to do it.

A simple solution may be what you initially mentioned. We can replace the variable "help needed on quarantine" with a more general variable  "cases needs" having multiple category options, each option followed by a free text:
cases need:
Immediate RT-PCR prescription
D+7 RT-PCR prescription
Medical absence from work certificate
Other(s)
If this last solution is implemented, then we will not be able to link the prescriptions to the samples and lab results that were documented as a result of this prescription.

**Comment by carolinverset** [comment]
@bernardsilenou 
sorry for the late response. I agree with your proposals.
Last question, is it possible to add the need for masks in the drop-down list? 
Are you ok to add a free field to specify the "other" modality? For cases and contacts? 
If ok, we send our developers to start the work. We can talk about it again at our 1 p.m. meeting

**Comment by bernardsilenou** [comment]
@carolinverset  If we go with this solution, then we can add mask and any other needs. However there is a request from other countries that is similar with this one. I will like to have a look at the other request and discuss it with the team before we begin with implementation. Hopefully by Friday.
